test/distributed/fsdp/test_fsdp_overlap.py

```diff
@@ -93,9 +93,9 @@ def world_size(self):
     def _dist_train(self):
         rank = self.rank
         world_size = self.world_size
```

Comment:
This test is disabled in CI, but I found that it is broken when running locally. This change fixes the test.